__tests__/app.test.js

```diff
@@ -178,4 +178,75 @@ describe('4. GET /api/articles/:article_id/comments', () => {
                 })
         })
     })
+})
+
+describe('5. POST /api/articles/:article_id/comments', () => {
+    describe('a. status 201 & data', () => {
+        it('returns 201 status code and the object of the comment back to the user', () => {
+            return db.query("INSERT INTO users (username, name) VALUES ('jxnbox', 'jan');")

```

Comment:
I would be hesitant to insert raw SQL into your test in this manner. You are messing with the database without using a tested endpoint. It's fine currently and I admire the creativity but this could go wonky very quickly. It's also not very dry because you have to insert this new user each test.

If you made a postUser endpoint and used that to add a new valid user then posted a comment with that valid user, I wouldn't have an issue 👍 

I would look in the users table and only test with users that are already present. You could use "jxnbox" for your invalid username test I've suggested lower down 😀